upb_generator/bootstrap_compiler.bzl

```diff
@@ -67,7 +67,7 @@ def _stage0_proto_staleness_test(name, base_dir, src_files, src_rules, strip_pre
             "$(location " + _protoc + ") " +
             "-I$(GENDIR)/" + strip_prefix + " " + _extra_proto_path +
             "--plugin=protoc-gen-upb=$(location " + _upbc("upb", 0) + ") " +
-            "--upb_out=bootstrap_upb:$(@D)/bootstrap_generated_sources/" + base_dir + "stage0 " +
+	    "--upb_out=dllexport_decl=TEST_DLL,bootstrap_upb:$(@D)/bootstrap_generated_sources/" + base_dir + "stage0 " +
```

Comment:
Hm, can you explain this part of the change to me? Or is this just lingering from testing the flag?

Reply:
This was put in as a way of smoke-testing the dllexport_decl parameter - by invoking it as part of the build we at least get a failure if it's totally broken. However, I now realize that since removing the 'export declarations' from the generated files (reverting to 'option 1' from 'option 2'), this will likely break the build because the TEST_DLL symbol is no longer being explicitly defined.

Can you suggest a place I can add a proper test for this?

Reply:
We unfortunately don't have a good place for this shape of codegen integration test, in the interest of moving forward we can leave this without a test and hopefully get coverage later. Thanks!